modules/openthread/platform/crypto_psa.c

```diff
@@ -82,6 +86,10 @@ static psa_key_usage_t toPsaKeyUsage(int aUsage)
 		usage |= PSA_KEY_USAGE_SIGN_HASH;
 	}
 
+	if (aUsage & OT_CRYPTO_KEY_USAGE_VERIFY_HASH) {
+		usage |= PSA_KEY_USAGE_VERIFY_HASH;
```

Comment:
I guess `checkKeyUsage` below must be updated as well

Reply:
fixed.